src/main/java/frc/robot/subsystems/ColorSensorSubsystem.java

```diff
@@ -49,7 +49,7 @@ private void MuxChangeI2cPort(int newMuxPort) {
     // number to that port
     boolean failed = I2CObject.write(i2cPortId, newMuxPort);
     if (failed) {
-      System.out.println("Failed to write to MUX over I2C");
```

Comment:
Why remove the error statement? If something is going wrong then it should be writing notifications to the log, no?

Reply:
just leftover from trying to chase down what was causing a loop overrun, we shouldn't be printing like that anyway i thought

Reply:
In this case it's pretty well supported, even encouraged since it logs to the RioLog. If you're worried about the performance you can have it only ever print the error once before silently failing every time afterwards.